packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -481,17 +416,27 @@ void _applyCameraUpdate(gmaps.GMap map, CameraUpdate update) {
 
 // original JS by: Byron Singh (https://stackoverflow.com/a/30541162)
 gmaps.LatLng _pixelToLatLng(gmaps.GMap map, int x, int y) {
-  final ne = map.bounds.northEast;
-  final sw = map.bounds.southWest;
+  final bounds = map.bounds;
   final projection = map.projection;
+  final zoom = map.zoom;
+
+  assert(
+      bounds != null, 'Map Bounds required to compute LatLng of screen x/y.');
+  assert(projection != null,
+      'Map Projection required to compute LatLng of screen x/y');
+  assert(zoom != null,
+      'Current map zoom level required to compute LatLng of screen x/y');
+
+  final ne = bounds!.northEast;
+  final sw = bounds.southWest;
 
-  final topRight = projection.fromLatLngToPoint(ne);
-  final bottomLeft = projection.fromLatLngToPoint(sw);
+  final topRight = projection!.fromLatLngToPoint!(ne)!;
+  final bottomLeft = projection.fromLatLngToPoint!(sw)!;
```

Comment:
I think a comment here about why this is safe would be useful.

Reply:
(Asserts to the rescue)